training_configs/vilbert_vqa_pretrained.jsonnet

```
@@ -54,7 +53,7 @@ local vocabulary = if construct_vocab then {
   "model": {
     "type": "vqa_vilbert_from_huggingface",
     "model_name": model_name,
-    "image_feature_dim": 1024,
+    "image_feature_dim": 2048,
```

```suggestion
    "image_feature_dim": 1024,
```
This wasn't supposed to be changed.